src/content/api/logging.md

```diff
@@ -26,6 +27,36 @@ By introducing webpack logging API we hope to unify the way webpack plugins and
 
 W> __Avoid noise in the log!__ Keep in mind that multiple plugins and loaders are used together. Loaders are usually processing multiple files and are invoked for every file. Choose a logging level as low as possible to keep the log output informative.
 
+__my-webpack-plugin.js__
```

Comment:
For SEO reasons i suggest adding a heading like:
Examples of how to get and use logger and infrastructureLogger in webpack loaders and plugins

Reply:
Added one.